### ISSUE TITLE
[PW-6590] placeOrder mutation in GraphQl not working

### ISSUE THREAD

**Issue by marcoboom** [Adyen/adyen-magento2#1470]
**Describe the bug**
Since the latest 8.2.1. release, the placeOrder GraphQl mutation is not working anymore when adding adyen_payment_status to the GraphQl query, for example:

```graphql
mutation ($input: PlaceOrderInput) {
  placeOrder(input: $input) {
    order {
      cart_id
      order_number
      adyen_payment_status {
        action
        additionalData
        isFinal
        resultCode
      } 
    }
  }
}
```

**To Reproduce**
Set an Adyen payment with the GraphQl mutation setPaymentMethodOnCart, after that place the order with the placeOrder mutation with the query as above.

Now receive the error on the frontend "Message: Unable to place order: Error with payment method, please select a different payment method., Location: undefined, Path: placeOrder".

In the logs you van see the following error:

```
[2022-04-22 09:07:01] AdyenLoggerTest.WARNING: Attempted to get the payment status for order 000000018. Exception: The cart isn't active. {"is_exception":false} []
```

The reason for this is that the PaymentResponseHandler disables the quote within the placeOrder process. You can find this at: https://github.com/Adyen/adyen-magento2/blob/8.2.1/Helper/PaymentResponseHandler.php#L225

But next when you add adyen_payment_status to the response of the GraphQl mutation, the resolver GetAdyenPaymentStatus is called. That file wants to fetch the quote / cart with the getCartForUser dependency. That method provides a check if the cart is active, if not, a GraphQl exception is thrown. You can find this code in https://github.com/Adyen/adyen-magento2/blob/8.2.1/Model/Resolver/GetAdyenPaymentStatus.php#L108

**Expected behavior**
Do not use the getCartForUser dependency in Model\Resolver\GetAdyenPaymentStatus but make an own check for this because the cart is mostly inactive at this point.

**Magento version**
2.4.3-p1

**Plugin version**
8.2.1

**Comment by tnaber** [comment]
Hi @marcoboom, @marc-arnoult and @jesse-deboer,

Thank you for opening this issue. I have opened an internal ticket for this and we will start working on this asap.

Cheers,
Titus

**Comment by Morerice** [comment]
Fix was released in v8.2.2
